### ISSUE TITLE
preparations for js-ipfs 0.36

### ISSUE THREAD

**Issue by alanshaw** [peer-base/peer-base#285]
👋 I upgraded `ipfs` to `0.36.0-rc.0` and some of the tests failed 😢. Please check out the [release issue](https://github.com/ipfs/js-ipfs/issues/2024) for info on breaking changes.

It looks as though the code is expecting a `_floodSub` property to be available on the libp2p node but I think it has moved!

**Comment by jimpick** [comment]
Yes, peer-base is stuck a few versions behind. It didn't get updated for a few months when it's fate was uncertain, and so it will require some attention. It is unlikely to get an update until after IPFS Camp - it's not an OKR for Q2. It probably would be best to remove it as a release criteria for the moment.
